src/Command/MarkMigrated.php

```diff
@@ -60,14 +61,33 @@ protected function configure()
             ));
         $this->addOption('plugin', 'p', InputArgument::OPTIONAL, 'The plugin the file should be created for')
             ->addOption('connection', 'c', InputArgument::OPTIONAL, 'The datasource connection to use')
-            ->addOption('source', 's', InputArgument::OPTIONAL, 'The folder where migrations are in');
+            ->addOption('source', 's', InputArgument::OPTIONAL, 'The folder where migrations are in')
+            ->addOption(
+                'target',
+                'u',
+                InputArgument::OPTIONAL,
+                'If present it will mark migrations from beginning to the given one'
+            )
+            ->addOption(
+                'exclude',
+                't',
```

Comment:
Now I got it. That sounds cool :)

I will wait for any suggestion from @dakota and then I implement it in this way.

@HavokInspiration no problem about that. It was worth it!